### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by kevindeedavis**

### CODE REVIEW

File: skins/modal/index.ess [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.Modal
```
Comment (camshaft) [inline skins/modal/index.ess]: let's remove this selector and dedent the whole thing. that way the developer has control over the selector name

File: skins/figures/index.ess [hunk 2]
```diff
@@ -1,2 +1,56 @@
...
+  font-weight bold
+  left 50%
+  margin 1px 0 0 0
+  position absolute
+  text-transform uppercase
+  top 50%
+  transform translate(-50%, -50%)
+
+// avatars
+.FiguresAvatar
```
Comment (camshaft) [inline skins/figures/index.ess]: i feel like this should be in `skins/figures/avatar.ess`

File: skins/figures/index.ess [hunk 3]
```diff
@@ -1,2 +1,56 @@
...
+  
+  &-content
+    background-color = __.color.white
+    background-position center center
+    background-size cover
+    border-radius 1000px
+    padding-bottom 100%
+  
+// 16/9
+.Figures-16-9
```
Comment (camshaft) [inline skins/figures/index.ess]: same with this: `skins/figures/16-9.ess`

File: skins/figures/index.ess (hunk 2 above)
Comment (jaydenanderson) [inline skins/figures/index.ess]: Yeah. There's cleanup here we're going to do. The default figure will be figure avatar. It's in a separate PR that Kevin and I worked on together. We'll keep working on keeping our branches only related to the skins we're working on.

File: skins/figures/index.ess (hunk 3 above)
Comment (jaydenanderson) [inline skins/figures/index.ess]: @camshaft this is will be fixed as well.

File: skins/modal/index.ess (hunk 1 above)
Comment (jaydenanderson) [inline skins/modal/index.ess]: @camshaft yep.

**Pull request merged**